prog-o-meter.py

```diff
@@ -144,13 +166,115 @@ def add_day(self):
-
+        if self.days <= 0:  # Disable Whoops button if days become 0
+            self.whoops_button.config(state = "disabled")
+        # Enable add_day_button if days of progress is < goal
+        if self.add_day_button["state"] == "disabled":
+            self.add_day_button["state"] = "normal"
+        if self.add_multidays_button["state"] == "disabled":
+            self.add_multidays_button["state"] = "normal"
+    def add_multidays(self, days, goal, filename, username):
+        """Fill out multiple rectangle in prog-o-meter with blue color, to represent multiple days completed.
+        
+           Opens a window, which lets the user enter number of days they want to add to the progress bar.
+        """
+        self.root.destroy()
+        MultipleDaysGUI(days, goal, filename, username)
+        
+class MultipleDaysGUI(object):
+    """Class contains everything related to the user providing multiple days to increase their progress at a time
+       instead of pressing 1 more day! button multiple times.
+       
+     root: a tkinter root.
+        filename: the name of the .txt file storing the user's number of days.
+        username: the name of the user.
+        days: the number of days the user have completed.
+        goal: The number of days the user is trying to complete (hardcoded to be 100, but might wanna add feature for user to chooses this themselves in the future).
+    """
+    
+    def __init__(self, _days, goal, filename, username):
+        """
+        Opens a Tkinter window for the user to enter the number of days they want to add to the prog-o-meter.
+        Window contains a submit button, for user to click when they have entered the number of days.
+        """
+        # Attributes
+        self.root = Tk.Tk()
+        self.days = _days
+        self.goal = goal
+        self.filename = filename
+        self.username = username
+        # Tkinter instantiation
+        self.canvas_layout()
+        self.submit_button()
+        self.root.mainloop()
+        
+    def canvas_layout(self):
+        """Display a Tkinter canvas.
+        
+        Creates a 300x50 px canvas with a greeting text and an entry widget (input field for text).
+        
+        Attributes:
+            CANVAS_WIDTH: The width of the canvas is hardcoded to 300 px.
+            CANVAS_HEIGHT: The height of the canvas is hardcoded to 50 px.
+            canvas: The Tkinter canvas widget.
+            text_entry: A Tkinter text-entry widget (input field for text)
+        """
+        self.CANVAS_WIDTH = 300
+        self.CANVAS_HEIGHT = 50
+        self.canvas = Tk.Canvas(self.root, width = self.CANVAS_WIDTH, height = self.CANVAS_HEIGHT)
+        self.canvas.pack()
+        self.text_entry = Tk.Entry(self.root)
+        self.text_entry.pack()
+        self.text_entry.focus_force()
+        self.canvas.create_text(self.CANVAS_WIDTH/2, 20, text = "Enter the days you want to progress")

```

Comment:
I think maybe "Enter the days you want to add" is a little more clear to the user :)